devices/Ssd13xx/Ssd13xx.cs

```diff
@@ -145,6 +145,72 @@ protected SpanByte SliceGenericBuffer(int start, int length)
             return _genericBuffer;
         }
 
+        /// <summary>
+        /// Copies buffer content directly to display buffer.
+        /// Y and height must be byte aligned because buffer will 
+        /// be copied without any logical operations on existing content.
+        /// </summary>
+        /// <param name="x">The x coordinate on the screen.</param>
+        /// <param name="y">The y coordinate on the screen.</param>
+        /// <param name="width">Width of buffer content in pixels.</param>
+        /// <param name="height">Height of buffer content in pixels.</param>
+        /// <param name="buffer">Data to copy.</param>
+        public void DrawDirectAligned(int x, int y, int width, int height, byte[] buffer)
+        {
+            if ((y % 8) != 0)
+            {
+                throw new ArgumentException("Y must be aligned to byte boundary.");
+            }
+            if ((height % 8) != 0)
+            {
+                throw new ArgumentException("Height must be aligned to byte boundary.");
+            }
+
+            var dataHeightInBytes = height / 8;
+            if ((dataHeightInBytes * width) != buffer.Length)
+            {
+                throw new ArgumentException("Width and height do not match the bitmap size.");
+            }
+
+            var genericBufferIdx = y / 8 * Width + x;
+            var srcIdx = 0;
+            for (int i=0; i<dataHeightInBytes; i++)
```

Comment:
please use `ctrl + k + d` in Visual Studio to have a proper alignment of the code ;-)